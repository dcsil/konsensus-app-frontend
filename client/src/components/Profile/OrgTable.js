import { Table } from 'reactstrap';
// core components
import OrgRow from './OrgRow';

const OrgTable = ({ firstName, lastName, email, role }) => {
  return (
    <Table className="align-items-center table-flush" responsive>
      <thead className="thead-light">
        <tr>
          <th scope="col" />
          <th scope="col">Name</th>
          <th scope="col">Role</th>
          <th scope="col" />
        </tr>
      </thead>
      <tbody>
        <OrgRow
          firstName={firstName}
          lastName={lastName}
          email={email}
          role={role}
        />
      </tbody>
    </Table>
  );
};

export default OrgTable;
