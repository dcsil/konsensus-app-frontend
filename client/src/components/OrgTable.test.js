import React from 'react';
import { shallow } from 'enzyme';
import OrgTable from './Profile/OrgTable';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

test('Test OrgTable component', () => {
  const wrapper = shallow(<OrgTable />);
  expect(wrapper).toMatchSnapshot();
});
