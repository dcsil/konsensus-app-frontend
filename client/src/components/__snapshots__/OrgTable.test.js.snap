// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test OrgTable component 1`] = `ShallowWrapper {}`;
